incubator/kubernetes_spark_install/preparation/script.md

```diff
@@ -121,4 +165,16 @@ k8s-agent-b90bd903-0    Ready                      1h        v1.5.3
 k8s-agent-b90bd903-1    Ready                      1h        v1.5.3
 k8s-agent-b90bd903-2    Ready                      1h        v1.5.3
 k8s-master-b90bd903-0   Ready,SchedulingDisabled   1h        v1.5.3
-```
\ No newline at end of file
+```
+
+## Get the Kubernetes Spark Examples
+
+The Kubernetes [GitHub repository](https://github.com/kubernetes/kubernetes) includes helpful examples to get started. One of which, is a set of configuration files for Spark (`examples/spark`).
+
+Let's clone the repository so that we can install Spark and Zeppelin easier later on.
+
+```
+git clone https://github.com/kubernetes/kubernetes.git

```

Comment:
This will clone Kubernetes into the current directory, which (if working from acs-demos source) will be under version control. Will this cause issues under Git?

Furthermore, and more certainly, the script.md file assumes that the Kubernetes source is in `~/Kubernetes` and therefore I suspect this will break.